test/REQUIRE

```diff
@@ -0,0 +1,2 @@
+ImageCore 0.7
+MAT 0.4+
```

Comment:
is the trailing `+` a typo or on purpose? Have not seen one before

Reply:
I put it on purpose as a reminder that MAT v0.4 is not compatible with Julia v1.0 and it should be updated as soon as the new version of MAT is released.
Julia parses it, but IIUC it doesn't conform to semantic versioning specification, so I removed it to avoid problems with METADATA.jl CI tests.